docs/data/whats-new/WhatsNewLayout.tsx

```diff
@@ -20,6 +20,39 @@ type Blog = {
 };
 
 const blogs: Blog[] = [
+  {
+    title: 'MUI X v6.18.0',
+    description:
+      'New stable components, polished features, better performance and more.',
+    announcementDate: 'Monday, Nov 13, 2023',
+    url: 'https://mui.com/blog/mui-x-end-v6-features/',
+    highlightList: [
+      {
+        title: 'Charts - stable version',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#charts',
+      },
+      {
+        title: 'Tree view - stable version',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#tree-view',
+      },
+      {
+        title: 'Date and time clearable fields',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#clearable-field',
+      },
+      {
+        title: 'Customization Playgrounds for date and time pickers',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#customization-playgrounds',
+      },
+      {
+        title: 'DataGrid Column Autosizing',
```

Comment:
```suggestion
        title: 'DataGrid column autosizing',
```

Not sure we should capitalize those words
cc @samuelsycamore

Reply:
Yeah, it'd be good to clarify, @samuelsycamore; I treated it as a [proper noun](https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?pvs=4#3b58a114caf34e34a6c7918cbab8eeb7) considering it's a feature name, but I'm unsure since it's name is basically a description of behavior.

```suggestion
        title: 'Data Grid column auto-sizing',
```